src/exposurelog/app.py

```diff
@@ -3,17 +3,32 @@
 import fastapi
 
 from . import shared_state
-from .routers import add_message, delete_messages, edit_message, find_messages
+from .routers import (
+    add_message,
+    delete_message,
+    edit_message,
+    find_messages,
+    get_message,
+)
 
 app = fastapi.FastAPI()
 
-app.include_router(add_message.router)
-app.include_router(delete_messages.router)
-app.include_router(edit_message.router)
-app.include_router(find_messages.router)
+subapp = fastapi.FastAPI(
+    title="Exposure log service",
+    description="A REST web service to create and manage log messages "
+    "that are associated with a particular exposure. "
+    "It is self-documenting using OpenAPI.",
```

Comment:
```suggestion
    description="A REST web service to create and manage log messages "
    "that are associated with a particular exposure. ",
```

Reply:
I would prefer that this match the README. So would prefer to change neither or both. See my comment above about your suggested change to the README.

Reply:
If anything, it'd be useful in this context for the root public GET endpoint to include in its body the URL to the OpenAPI docs.

Reply:
Good suggestion. Done.